webviz_config/webviz_deprecated.py

```diff
@@ -0,0 +1,85 @@
+from typing import Dict, Union, Callable, Tuple, cast, Optional, List
```

Comment:
Rename this to something such that we don't repeat `webviz`? E.g. `webviz_config.deprecation_decorators`.